Lib/calendar.py

```diff
@@ -510,9 +530,9 @@ def formatmonthname(self, theyear, themonth, withyear=True):
         """
         _validate_month(themonth)
         if withyear:
-            s = '%s %s' % (month_name[themonth], theyear)
+            s = '%s %s' % (alt_month_name[themonth], theyear)
         else:
-            s = '%s' % month_name[themonth]
+            s = '%s' % alt_month_name[themonth]
```

Comment:
What's the difference between this and `str(alt_month_name[themonth])`. And also, is it possible to have non-strings in `alt_month_name`?